extensions/BugModal/web/new_bug.js

```diff
@@ -38,17 +38,31 @@ $(document).ready(function() {
         options: [],
     });
 
+    var keywords_sel = $("#keywords").selectize({
+        delimiter: ', ',
+        valueField: 'name',
+        labelField: 'name',
+        searchField: 'name',
+        options: [],
+        preload: true,
+        create: false,
+        load: function(query, callback) {
+            callback(initial.keywords);
+        }
+    });
+
     product_sel.on("change", function () {
         $('#product-throbber').show();
         $('#component').attr('disabled', true);
+        $('#version').attr('disabled', true);
```

Comment:
Yes, again a mistake. I'll change it. Anyways it doesn't make any UI change as selectize doesn't seem to heed to that instruction.